src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -109,8 +109,8 @@
 
   <!-- Common dependencies across all targets -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.10.*" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.12.*" />
+	<PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.10.1" />
+	<PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.12.1" />
```

Comment:
Shouldn't it be unnecessary to replace `*` with `1` here?

Reply:
Also, the new package versions should be 1.13.0 and 2.11.0 because these aren't patch version changes, but larger feature changes. Incrementing the patch instead of the minor version broke our current CI build because it was picking up on suspend and resume packages. We should use 1.13.* and 2.11.*

Reply:
Done. Will probably have to re-publish to MyGet, no?

Reply:
Yes, the new versions will need to be published to MyGet. Let's do this once the durabletask changes are signed off by me.